spec/prog/download_boot_image_spec.rb

```diff
@@ -5,11 +5,13 @@
 RSpec.describe Prog::DownloadBootImage do
```

Comment:
I'd like to see this change in one commit. If you prefer two commits, please ensure they're consecutive. Same for the other test commit too.

```
Move default image versions from host_nexus to download_boot_image
Add tests moving image version from host_nexus to download_boot_image
```